apps/storybook-react-native/.storybook/storybook.requires.js

```diff
@@ -6,30 +6,30 @@ import {
   addParameters,
```

Comment:
@brianacnguyen do we want to ignore this file too or should we ensure prettier runs when it is generated?